hydromt_wflow/wflow.py

```diff
@@ -3533,9 +3537,9 @@ def write_forcing(
                     f"Not all dates found in precip_fn changing starttime to \
 {start} and endtime to {end} in the toml."
                 )
-                self.set_config("starttime", start.to_pydatetime())
-                self.set_config("endtime", end.to_pydatetime())
-                self.write_config()
+                # Set the strings first
+                self.set_config("starttime", start.strftime("%Y-%m-%dT%H:%M:%S"))
+                self.set_config("endtime", end.strftime("%Y-%m-%dT%H:%M:%S"))
```

Comment:
Why did you change to strings? Wflow can handle both or did you have some errors?

Reply:
More uniform handling of the date typing. It now ran from string to datetime, sometimes back.